docs/manuals/source/Appendix/Debugging.rst

```diff
@@ -168,16 +168,38 @@ Our Linux packages are stripped of debugging symbols, so you need an extra step
 
 
    .. code-block:: shell-session
-      :caption: Installing Bareos debug symbols on (RH)EL system
+      :caption: Installing Bareos Director debug symbols on (RH)EL system
 
-      dnf --enablerepo bareos-debuginfo install bareos-director-debuginfo
+      dnf --enablerepo=bareos-debuginfo install gdb bareos-debuginfo bareos-director-debuginfo \
+      bareos-database-common-debuginfo bareos-common-debuginfo bareos-database-tools-debuginfo
 
    .. code-block:: shell-session
-      :caption: Installing Bareos debug symbols on (open)SUSE system
+      :caption: Installing Bareos Director debug symbols on (open)SUSE system
 
       # Enable and activate bareos-debuginfo repository
       zypper modifyrepo --enable --refresh --gpgcheck bareos-debuginfo
 
-      zypper install bareos-director-debuginfo
+      zypper install gdb bareos-debuginfo bareos-director-debuginfo bareos-common-debuginfo \
+      bareos-database-common-debuginfo bareos-database-tools-debuginfo
+
+.. note::
+   For rpm: you maybe want to install all corresponding **-debuginfo** of installed **bareos-**
+   packages if you want to debug all.
```

Comment:
Maybe we want to mention that you can do so with
```
dnf debuginfo-install bareos-*-debuginfo
```
zypper probably works similarly.

Reply:
Nice improvement for dnf call I will use that one.
and thank to force myself to go further, zypper has something similar :-)
```
zypper -v --plus-content bareos-debuginfo install -r bareos-debuginfo bareos-*-debuginfo
```
will use that too.

Reply:
done